steward/src/cellars.rs

```diff
@@ -17,7 +18,19 @@ impl std::fmt::Display for CellarId {
 }
 
 pub async fn get_gas_price() -> Result<U256, Error> {
-    CellarGas::etherscan_standard().await.map_err(|e| e.into())
+    if std::env::var("ETHERSCAN_API_KEY").is_ok() {
+        match CellarGas::etherscan_standard().await {
```

Comment:
If we have any desire to run against anything other than Ethereum mainnet, we might also want to port the chain lookup code from the bridge as well:

https://github.com/PeggyJV/gravity-bridge/blob/fef4573e412be03a622496b0bd24365dd95265b8/orchestrator/ethereum_gravity/src/utils.rs#L141-L177